### ISSUE TITLE
Live Share for server

### ISSUE THREAD

**Issue by hajile-arch** [microsoft/vscode#233280]
Type: <b>Bug</b>

I am using react vite to live share, I can live share everything and even the server. but when my friend press the server it keeps on loading forever. 

VS Code version: Code 1.95.1 (65edc4939843c90c34d61f4ce11704f09d3e5cb6, 2024-10-31T05:14:54.222Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 4600H with Radeon Graphics          (12 x 2994)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|23.36GB (6.23GB free)|
|Process Argv|--open-url --crash-reporter-id 99fb2833-a7bb-40bc-ae21-aa4a15a49646 -- vscode://ms-vsliveshare.vsliveshare/join?vslsLink=https://prod.liveshare.vsengsaas.visualstudio.com/join%253F69803D2EE1F88954007C12015912903CCC0D|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (49)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-django|bat|1.15.0
vscode-intelephense-client|bme|1.12.6
vscode-tailwindcss|bra|0.12.12
path-intellisense|chr|2.9.0
c-cpp-compile-run|dan|1.0.60
python-environment-manager|don|1.2.4
python-extension-pack|don|1.7.0
es7-react-js-snippets|dsz|4.4.3
vscode-html-css|ecm|2.0.11
prettier-vscode|esb|11.0.0
c-cpp-runner|fra|9.4.9
kotlin|fwc|0.2.35
live-sass|gle|6.1.2
vsc-python-indent|Kev|1.18.0
Kotlin|mat|1.7.1
mongodb-vscode|mon|1.9.3
debugpy|ms-|2024.12.0
isort|ms-|2023.10.1
python|ms-|2024.18.0
vscode-pylance|ms-|2024.11.1
jupyter|ms-|2024.10.0
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.21
vscode-jupyter-cell-tags|ms-|0.1.9
vscode-jupyter-slideshow|ms-|0.1.6
cpptools|ms-|1.22.11
vscode-typescript-next|ms-|5.8.20241106
vsliveshare|ms-|1.0.5941
live-server-preview|neg|0.1.4
autodocstring|njp|0.6.1
material-icon-theme|PKi|5.13.0
java|red|1.36.0
r|REd|2.8.4
LiveServer|rit|5.7.9
es7-react-js-snippets|rod|1.9.3
iconfont-preview|stx|0.0.5
typewriter-sounds|tim|0.1.0
vscode-pets|ton|1.27.0
vscode-lldb|vad|1.11.1
vscode-gradle|vsc|3.16.4
vscode-java-debug|vsc|0.58.1
vscode-java-dependency|vsc|0.24.1
vscode-java-pack|vsc|0.29.0
vscode-java-test|vsc|0.42.0
vscode-maven|vsc|0.44.0
jinja|who|0.0.8
JavaScriptSnippets|xab|1.8.0
php-debug|xde|1.35.0
html-css-class-completion|Zig|1.20.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
jg8ic977:31013176
dvdeprecation:31068756
dwnewjupyter:31046869
impr_priority:31102340
nativerepl2:31139839
refactort:31108082
pythonrstrctxt:31112756
cf971741:31144450
iacca1:31171482
notype1cf:31157160
5fd0e150:31155592
dwcopilot:31170013

```

</details>

**Comment by hajile-arch** [comment]
so I am using this framework, vite and then inside vite I am using react
and typescript and node.js to do my project. So when I typed npm run dev.
it will show something like this
[image: image.png]

At the live share's available functions, there is one part where we can
share server. But when we put in our link which for example is
http://localhost:5173/ (as shown below), and when the person we shared it
to, clicked it after we insert, it will keep on loading on their browser
page. We have tried this with other internet and network and it
isn't working.

[image: image.png]
